cluster/ci/config/prow/openshift/config_updater_rbac.yaml

```diff
@@ -73,6 +92,24 @@ items:
         - create
         - update
         - patch
+    - apiGroups:
```

Comment:
I think this is the surprising one -- what needs to change RBAC in `ci` namespace? We need to be wary of anything that could be back-dooring or otherwise hosing our production installs.